libs/styles/styles.css

```diff
@@ -345,6 +345,14 @@
   width: 100%;
 }
 
+.xxl-button .con-button {
```

Comment:
the button classes usually have the button first, i.e. `button-l`, `button-xl`, may want to follow that pattern?

Reply:
`.xxl-button` is what Ryan Clayton, Ryan Parrish, and I agreed on. The reasoning is because that follows the pattern for other block variants `.xs-spacing`, `.xxs-gap`, `.l-heading`, etc